watch/watch.1

```diff
@@ -7,41 +7,54 @@
 .SH NAME
 watch \- Keep an eye on a command output.
 .SH SYNOPSIS
-\fBwatch\fR \fB\-n\fR [\fIseconds\fR] [\fB\-bt\fR] [\fBcommand\fR [\fIargs\fR...]]
+\fBwatch\fR [\fB\-n\fR \fIseconds\fR] [\fB\-bt\fR] \fBcommand\fR [\fIargs\fR...]
 .SH DESCRIPTION
 The
 .B watch 
-utility executes a command repeatedly every "user-defined" seconds.
-This allows you to watch a program output change over the time.
+utility executes a command repeatedly every
+"user-defined" seconds. This allows you to watch
+a program output change over the time.
 .PP
 It accepts the following options:
 .TP
 .B \-n
-Specifies the interval to refresh the output. Per default, the
-interval is of 2 seconds.
+Specifies the interval to refresh the output.
+Per default, the interval is of 2 seconds.
 .TP
 .B \-b
-Beeps if the program being run returns a non-zero exit status.
+Beeps if the program being run returns a non-zero
+exit status.
 .TP
 .B \-t
-Turns off the information header, letting just the verbatim program
-output being refreshed every time on the screen \(em equivalent to a \fIksh\fR(1)
+Turns off the information header, letting just the
+verbatim program output being refreshed every time
+on the screen \(em equivalent to a \fIksh\fR(1)
 \fIfor ((;;))\fR accompanied by \fIsleep\fR(1).
 .br
-Not recommended, unless you're using \fItmux\fR(1) or another terminal multiplexer
-that shows you information that would originally be shown by that header.
+Not recommended, unless you're using \fItmux\fR(1)
+or another terminal multiplexer that shows you
+information that would originally be shown by that
+header.
 .SH EXAMPLES
-It can be very useful when backuping some disc and needing to see how many files
-were already copied.
+It can be very useful when backuping some disc and
+needing to see how many files were already copied.
 .br
-Lets suppose you're copying a disc from a terminal and, in another, you want to
-keep an eye at \fIdu\fR(1) output to see every minute if the file size matches
-with what you're expecting.
+Lets suppose you're copying a disc from a terminal
```

Comment:
```suggestion
Let's suppose you're copying a disk from a terminal
```
Not trying to be nitpicky about this, but this is quite informal and also a bit convoluted.
I would recommend something along the lines of
```
To automatically check a file's size every 60 seconds:
[insert your command here]
```
Let me know your feedback.

Reply:
Note that I changed `disc` to `disk` because the former refers to optical media only whilst the latter refers to any digital media.

Reply:
Seems good, I will commit those suggestions.

Reply:
> Not trying to be nitpicky about this, but this is quite informal and also a bit convoluted.

Understandable. Your replacement suggestion sounds better.